### DISCUSSION TITLE
Dismiss a floating element by clicking itself

### DISCUSSION THREAD

**Discussion by annmarie-switzer** [floating-ui/floating-ui#2436] [Q&A] [answered]
Is there a way to dismiss a floating element by clicking within itself? For example, I have a popover with a button inside of it. I want users to be able to click this "close" button to dismiss the popover (in addition to an off-click of esc key press). Is this currently possible?

**Comment by atomiks** [comment, marked as answer]
The `isOpen` state can be freely controlled by you - the reusable example should show this:
```js
<button onClick={() => setIsOpen(false)}>close</button>
```

**Reply by annmarie-switzer** [reply]
Thank you so much!
